nmrfx-core/src/main/java/org/nmrfx/utilities/ByteConversion.java

```diff
@@ -15,19 +15,14 @@
 public class ByteConversion {
 
     public static Number[] convert(byte[] buffer, String sMode, int start, int n) {
-        boolean littleEndian = true;
         ByteBuffer byteBuffer = ByteBuffer.wrap(buffer);
         Number[] numbers = new Number[n];
         int offset = start;
         for (int i = 0; i < n; i++) {
             if (sMode.equals("b")) {
                 numbers[i] = byteBuffer.get(offset);
             } else {
-                if (littleEndian) {
-                    byteBuffer.order(ByteOrder.BIG_ENDIAN);
-                } else {
-                    byteBuffer.order(ByteOrder.LITTLE_ENDIAN);
-                }
+                byteBuffer.order(ByteOrder.BIG_ENDIAN);
```

Comment:
Asked @brucejohnson to confirm that we only need to use BIG_ENDIAN here so removed if/else